app/styles/modules/_settings.scss

```diff
@@ -573,3 +573,13 @@ section.modal-panel {
     margin-bottom: 30px;
   }
 }
+
+.fxa-radio {
+  text-align: left;
```

Comment:
@divyabiyani with SCSS you can nest CSS rules, see http://sass-lang.com/guide#topic-3

We need to support RTL languages here as well, example: https://github.com/mozilla/fxa-content-server/blob/c3cc9d4a462e9c96cd702a5e0161065c4d8c0139/app/styles/modules/_tooltip.scss#L10-L16 

Let me know if you need help testing this...